package.json

```diff
@@ -25,7 +25,6 @@
     "@docusaurus/plugin-sitemap": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
     "@docusaurus/theme-classic": "2.2.0",
-    "@docusaurus/theme-search-algolia": "2.2.0",
```

Comment:
I'd like to leave this here -- Algolia is coming shortly

Reply:
It's unnecessary regardless, thus the removal! I'd love to have Algolia here—so much better. 

<img width="986" alt="Screen Shot 2022-11-20 at 3 31 09 PM" src="https://user-images.githubusercontent.com/10472920/202927204-f2faa424-a23b-4e80-bf74-f497b1920bbd.png">

Reply:
Oh awesome, thanks for confirming!